src/panel.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="content-type" content="text/html; charset=utf-8">
+    <title>Block-facebook</title>
```

Comment:
Should this be "Block-facebook" or "Facebook Container" (although possibly irrelevant if the title isn't visible anywhere)